examples/click-events-basic/readme.md

```diff
@@ -0,0 +1,115 @@
+# MongoDB Charts Embedding Example - Click Events 
+
+## Background
+
+⚠ _This example uses Beta functionality which is subject to change before the release. You must use the `@beta` tagged version of `@mongodb-js/charts-embed-dom` to use this functionality._
+
+🎮 _[Play with a live demo of this sample here](https://codesandbox.io/s/github/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-basic)_
+
+MongoDB Charts allows you to create visualizations of your MongoDB data using a simple web interface. You can view the visualizations within the Charts UI, or you can use the Embedding feature to render the charts in an external web application.
+
+When you embed charts using the Embedding SDK, you are able to subscribe to events that show when a user clicked on a chart, and the details of the element on which they clicked. This feature can be used to build interactive charts. This sample shows the basics of how to subscribe to and handle click events, including extracting relevant details about the chart elmement that was clicked. You can also see a more advanced demo that demonstrates [interactive filtering](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-filtering).
```

Comment:
Small suggestion: "subscribe to events that trigger" instead of "show". (that's for both readmes)